src/jj_terms.jl

```diff
@@ -45,14 +45,7 @@ function terms(orb::RelativisticOrbital, w::Int=one(Int))
     reverse!(_terms_jw(j, w))
 end
 
-function _terms_jw(j::HalfInteger, w::Integer)
-    j >= 0 || throw(DomainError(j, "j must be positive"))
-    1 <= w <= 2*j+1 || throw(DomainError(w, "w must be 1 <= w <= 2j+1 (=$(2j+1)) for j=$j"))
-    # This works by considering all possible n-particle fermionic product states (Slater
-    # determinants; i.e. each orbital can only appear once) of the orbitals with different
-    # m quantum numbers -- they are just different n-element combinations of the possible
-    # m quantum numbers (m ∈ -j:j).
-    #
+function _terms_jw_histogram(j, w)
     # Each of the product states is still a J_z eigenstate and the eigenvalue is just a sum
     # of the J_z eigenvalues of the orbitals. As for every coupled J, we also get M ∈ -J:J,
     # we can just look at the histogram of all the M quantum numbers to figure out which
```

Comment:
This comment should also move into the function below. It primarily applies to what comes after the `_terms_jw_histogram` call.